mongodbatlas/mongodbatlas.go

```diff
@@ -34,13 +34,11 @@ import (
 )
 
 const (
-	CloudURL        = "https://cloud.mongodb.com/"
-	defaultBaseURL  = CloudURL + APIPublicV1Path
-	APIPublicV1Path = "api/atlas/v1.0/" // APIPublicV1Path specifies the v1 api path
-	jsonMediaType   = "application/json"
-	plainMediaType  = "text/plain"
-	gzipMediaType   = "application/gzip"
-	libraryName     = "go-mongodbatlas"
+	defaultBaseURL = "https://cloud.mongodb.com/"
+	jsonMediaType  = "application/json"
+	plainMediaType = "text/plain"
+	gzipMediaType  = "application/gzip"
+	libraryName    = "go-mongodbatlas"
```

Comment:
I have removed `APIPublicV1Path` because we will use https://cloud.mongodb.com/ from now on

Reply:
you could've kept it and concatenate